scripts/installers/make-pyinstaller

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
```

Comment:
Can we call this script `make-exe`?  I'd rather name it after what it makes rather than the tool it uses.